website/addons/wiki/templates/edit.mako

```diff
@@ -333,6 +340,7 @@ ${parent.javascript_bottom()}
 
     window.contextVars = window.contextVars || {};
     window.contextVars.wiki = {
+        name: "${wiki_name}",
```

Comment:
I don't think you want to use ` | n`, as that may result in unsafe input shown onto the page.

Make sure you are not putting quotes around your `json.dumps` call.